index.js

```diff
@@ -1,6 +1,30 @@
 
 import { NativeModules } from 'react-native';
+import { Buffer } from 'buffer';
 
 const RNUportSigner = NativeModules.RNUportSignerModule;
 const RNUportHDSigner = NativeModules.RNUportHDSignerModule;
-export { RNUportSigner, RNUportHDSigner };
+const DerivationPathRoot = RNUportHDSigner.UPORT_ROOT_DERIVATION_PATH;
+
+function getSignerForAddress(address, derivationPath = DerivationPathRoot, userPrompt = "") {
+  return async (data) => {
+    const { v, r, s } = await RNUportHDSigner.signJwt(address,
+      derivationPath,
+      Buffer.from(data).toString('base64'),
+      'Sign a JWT');
```

Comment:
user prompt should be used here instead of hardcoded string
```suggestion
      userPrompt);
```